### PR TITLE
Updated build generation for v2 package

### PR DESCRIPTION
**Pull request by wbreza**
Update build generation for v2 release.  Include some gulp build pipeline commands from `dev` branch.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,8 +57,5 @@
     "webpack-stream": "^3.1.0",
     "yargs": "^4.4.0"
   },
-  "dependencies": {},
-  "publishConfig": {
```
Comment (mattmazzola) [inline package.json]: It looks like the publishConfig section was removed during some merge.  I think this section should stay.

File: PowerBI.JavaScript.nuspec [hunk 2]
```diff
@@ -17,6 +17,7 @@
   <files>
     <file src="dist\powerbi.js" target="content\scripts" />
     <file src="dist\powerbi.min.js" target="content\scripts" />
+    <file src="dist\powerbi.js.map" target="content\scripts" />
     <file src="dist\powerbi.d.ts" target="content\scripts\typings" />
```
Comment (mattmazzola) [inline PowerBI.JavaScript.nuspec]: Is there a way to set this to include the dist directory so we can match how npm works and not have to worry about updating this?

There are more .d.ts files than jsut the powerbi.d.ts now

File: package.json (hunk 1 above)
Comment (wbreza) [inline package.json]: I removed it to remove the `beta` flag

File: PowerBI.JavaScript.nuspec (hunk 2 above)
Comment (wbreza) [inline PowerBI.JavaScript.nuspec]: Let's take this as a TODO after this PR.

**Pull request merged**